test/cli_network_test.go

```diff
@@ -342,15 +337,22 @@ func (suite *PouchNetworkSuite) TestNetworkPortMapping(c *check.C) {
 
 	command.PouchRun("pull", image).Assert(c, icmd.Success)
 	command.PouchRun("run", "-d",
-		"--name", funcname,
+		"--name", cname,
 		"-p", "9999:80",
 		image).Assert(c, icmd.Success)
+	defer DelContainerForceMultyTime(c, cname)
 
-	time.Sleep(10 * time.Second)
-	err := icmd.RunCommand("timeout", "5", "curl", "localhost:9999").Compare(expct)
-	c.Assert(err, check.IsNil)
+	success := false
+	for i := 0; i < 60; i++ {
```

Comment:
This will try 60 times? 
Will it be so long?
Just double check. @zhuangqh

Reply:
Often, this command would success in several tries. Try many times to make sure the network port mapping is fail....longest timeout is 6 minutes.